CMakeLists.txt

```diff
@@ -47,7 +47,8 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE debug)
 endif()
 
-set(CMAKE_C_FLAGS         "${CMAKE_C_FLAGS} -Wall -Wextra -std=gnu11")
+set(CMAKE_C_STANDARD 11)
```

Comment:
I am confused here. If you are retaining C99 compatibility, this should be `set(CMAKE_C_STANDARD 99)`.